catalog/dags/common/sql.py

```diff
@@ -27,6 +27,14 @@
 # https://airflow.apache.org/docs/apache-airflow-providers-postgres/stable/_api/airflow/providers/postgres/hooks/postgres/index.html#airflow.providers.postgres.hooks.postgres.PostgresHook.copy_expert # noqa
 
 
+def _single_value(cursor):
```

Comment:
Now that this is in a shared location and used in other modules, we can remove the underscore prefix, right?

```suggestion
def single_value(cursor):
```

(Requires updates in import locations too, of course)

Reply:
Agreed! I'm going to make a note to update this in the followup in #2678, because even as simple as this is I'd still want to fully retest.